File: packages/cli/tests/commands/template/fixtures/start-cmd/expected/python-sync/build_prod.py
from e2b import Template, default_build_logger, handle_build_error
from template import template


if __name__ == "__main__":
    try:
        Template.build(
            template,
            "start-cmd",
            cpu_count=2,
            memory_mb=1024,
            on_build_logs=default_build_logger(),
        )
    except Exception as err:
        handle_build_error(err)

File: packages/cli/tests/commands/template/fixtures/multi-stage/expected/typescript/build.prod.ts
import { Template, defaultBuildLogger, handleBuildError } from 'e2b'
import { template } from './template'

async function main() {
  await Template.build(template, 'multi-stage', {
    onBuildLogs: defaultBuildLogger(),
  });
}

main().catch(handleBuildError);